auto-scale-aws.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Restarts AWS instance and changes the volume
+# Volume will be changed to t2.small if current season is "Off season"
+# The volume will be t2.medium if another season is active.
+
+
+# Using this function to remove double quotes from strings
+remove_double_quotes() {
+    echo $(sed -e 's/^"//' -e 's/"$//' <<<"$1")
+}
+
+off_season="Off season"
+instance_type_micro="t2.micro"
+instance_type_small="t2.small"
+instance_id="i-0ae22e94380789482"
```

Comment:
Is it ok to expose instance id? Should be ok but to be safe we can expose it through github secrets instead

Reply:
Though of that aswell. Fixing this.